test/flux_calculator_tests.jl

```diff
@@ -155,107 +154,84 @@ for FT in (Float32, Float64)
+        boundary_space = TestHelper.create_space(FT)
+
+        params = (; surface_scheme = scheme, FT = FT)
+
+        # atmos
+        p = (;
+            energy_bc = zeros(boundary_space),
+            ρq_tot_bc = zeros(boundary_space),
+            uₕ_bc = ones(boundary_space),
+            z = ones(boundary_space),
+            z_sfc = zeros(boundary_space),
+            u = ones(boundary_space),
+            v = ones(boundary_space),
+        )
+        Y_init = (; ρ = ones(boundary_space) .* FT(1.2), T = ones(boundary_space) .* FT(310), q = zeros(boundary_space))
+        integrator = (; Y_init..., p = p)
+        atmos_sim = TestAtmos(params, nothing, integrator)
+
+        # ocean
+        p = (;
+            F_aero = zeros(boundary_space),
+            z0m = FT(0.01),
+            z0b = FT(0.01),
+            beta = ones(boundary_space),
+            α = ones(boundary_space) .* FT(0.5),
+            q = zeros(boundary_space),
+            Cd = FT(0.01),
+            Ch = FT(0.01),
+            area_fraction = ones(boundary_space) .* FT(0.5),
+        )
+        Y_init = (; T = ones(boundary_space) .* FT(300))
+        integrator = (; Y_init..., p = p)
+        ocean_sim = TestOcean(nothing, integrator)
+
+        # ocean
+        ocean_sim2 = TestOcean(nothing, integrator)
+
+        model_sims = (; atmos_sim, ocean_sim, ocean_sim2)
+
+        coupler_cache_names = [
+            :T_sfc,
+            :surface_direct_albedo,
+            :surface_diffuse_albedo,
+            :F_R_sfc,
+            :F_R_toa,
+            :P_liq,
+            :P_snow,
+            :P_net,
+            :F_turb_energy,
+            :F_turb_ρτxz,
+            :F_turb_ρτyz,
+            :F_turb_moisture,
+            :z0m_sfc,
+            :z0b_sfc,
+            :beta,
+            :q_sfc,
+        ]
+        fields = Interfacer.init_coupler_fields(FT, coupler_cache_names, boundary_space)
+
+        # calculate turbulent fluxes
+        thermo_params = get_thermo_params(atmos_sim)
+        FluxCalculator.partitioned_turbulent_fluxes!(model_sims, fields, boundary_space, scheme, thermo_params)
+
+        # calculating the fluxes twice ensures that no accumulation occurred (i.e. fluxes are reset to zero each time)
+        # TODO: this will need to be extended once flux accumulation is re-enabled
+        FluxCalculator.partitioned_turbulent_fluxes!(model_sims, fields, boundary_space, scheme, thermo_params)
+
+        windspeed = @. hypot(atmos_sim.integrator.p.u, atmos_sim.integrator.p.v)
+
+        thermo_params = get_thermo_params(atmos_sim)
+        thermo_state_int = Interfacer.get_field(atmos_sim, Val(:thermo_state_int))
+
+        surface_thermo_states = similar(thermo_state_int)
+        surface_thermo_states .= FluxCalculator.surface_thermo_state(ocean_sim, thermo_params, thermo_state_int)
+
+        # NOTE: This test is very weak! We should add more stringent tests
+        @test Array(parent(fields.F_turb_moisture))[1] ≈ FT(0)
```

Comment:
can we add similar checks for the momentum and energy fluxes?

Reply:
Is there a reason we can't add checks for the other fluxes?

Reply:
Do we know that they should be zero?